code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -805,4 +1087,7 @@
 	locked = FALSE
 	INVOKE_ASYNC(src, PROC_REF(open))
 
+/obj/structure/closet/open
+	opened = TRUE
```

Comment:
Did you forget the icon state?

Reply:
There is no sensible opened icon state for closets. the door is added only as an overlay for closets
![Screenshot (197)](https://user-images.githubusercontent.com/110812394/236461368-4c3e1f2a-2143-4d17-b3ba-68230cb06c29.png)

But for crates it's a full icon state
![Screenshot (198)](https://user-images.githubusercontent.com/110812394/236461591-1ef43207-1dec-4462-9195-46d28215cfc8.png)

Reply:
Ahhhh interesting!  Carry on then!